Fix: docker build error "repository name must be lowercase "

asalkeld opened the pull request:
2 important fixes here:

commit cbf1d1a5d2dca51073bc3c00d4334a4e8f27608c

    fix: Make sure that Ignores make their way to docker build
    
    The problem is that boxygen Ignore has no effect. So publish
    a new function "BuildIgnore" on the runtime interface which
    is used by the build to pass to the docker excludes.

commit fe0bbab29c0e7ecb16d6c93189a0d21f3c3f578d

    fix: Make sure docker image names are lowercase
    
    closes #172

davemooreuws commented on packages/plugins/azure/src/resources/entrypoint.ts:
```diff
@@ -13,35 +13,241 @@
+			const routeName = `${type}-${target}-route`;
+			const routeConfig = {
+				backendPool: {
+					// FIXME: Add above defined pool name here...
```
Remove the FIXME? Looks like it was done.

davemooreuws commented on packages/plugins/azure/src/resources/topic.ts:
```diff
@@ -56,14 +59,14 @@ export class NitricEventgridTopic extends pulumi.ComponentResource {
-				topicName,
-			}),
-		);
+	// public getSharedAccessKeys = (): pulumi.Output<[string, string]> => {
```
Should this commented out code be removed?

davemooreuws reviewed (changes requested):
Looks good to me, just two questions

asalkeld committed:
refactor: make the javascript and typescript files easier to diff

asalkeld committed:
fix: Make sure docker image names are lowercase

closes #172

asalkeld committed:
fix: Make sure that Ignores make their way to docker build


The problem is that boxygen Ignore has no effect. So publish
a new function "BuildIgnore" on the runtime interface which
is used by the build to pass to the docker excludes.
